src/ucs/profile/profile.c

```diff
@@ -611,3 +620,70 @@ void ucs_profile_global_cleanup()
     ucs_profile_check_active_threads();
     pthread_key_delete(ucs_profile_global_ctx.tls_key);
 }
+
+#ifdef HAVE_NVTX
```

Comment:
can we also add runtime env var to enable/disable nvtx? for performance reason
also to control its color